errorScaleCal/plugins/errorScaleCal.cc

```diff
@@ -125,11 +127,7 @@ errorScaleCal::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
   event_.luminosityBlockNumber = iEvent.id().luminosityBlock();
   event_.eventNumber           = iEvent.id().event();
 
-
-  edm::ESHandle<TransientTrackBuilder>            theB                ;
-  edm::ESHandle<GlobalTrackingGeometry>           theTrackingGeometry ;
-  iSetup.get<GlobalTrackingGeometryRecord>().get(theTrackingGeometry) ;
```

Comment:
@dzuolo could you also remove the include for the GlobalTrackingGeometry?
`#include "Geometry/Records/interface/GlobalTrackingGeometryRecord.h"`

Reply:
Removed in https://github.com/MilanoBicocca-pix/BeamSpotCalibration/pull/4/commits/dc72fb88814b7907a221b48aa4b2be4319749e2a